core/src/main/java/org/mobilitydata/gtfsvalidator/notice/DuplicateKeyNotice.java

```diff
@@ -15,13 +15,26 @@
  */
 package org.mobilitydata.gtfsvalidator.notice;
 
+import static org.mobilitydata.gtfsvalidator.notice.SeverityLevel.ERROR;
+
 import javax.annotation.Nullable;
+import org.mobilitydata.gtfsvalidator.annotation.GtfsValidationNotice;
+import org.mobilitydata.gtfsvalidator.annotation.GtfsValidationNotice.SectionRefs;
+import org.mobilitydata.gtfsvalidator.annotation.GtfsValidationNotice.UrlRef;
 
 /**
  * The values of the given key and rows are duplicates.
  *
  * <p>Severity: {@code SeverityLevel.ERROR}
  */
+@GtfsValidationNotice(
+    severity = ERROR,
+    sections = @SectionRefs({"file-requirements"}),
```

Comment:
Is it worthwhile having the section ref as enums?

Reply:
I did consider that option.  My thinking at the time was that keeping it as a string would allow easier addition of new section refs without a code change, but maybe in reality that is overblown, since by definition if you are referencing a new section in a @GtfsVaildationNotice, you are making a code change in the Notice file, so updating an additional enum wouldn't be too much additional work.

Thoughts?

Reply:
I'm concerned that we can have typos in the ref sections, and this can be rendered out to the documentation without notice.

Reply:
Fair enough.  Let me make a change in a separate PR and then update this one to use it (since this PR is an automated change).

Reply:
Actually, I take that back, I made that change locally.  So I've fixed it in this PR.  Please take another look.